packages/react-jsx-babel-plugin/src/TransformJSXToReactBabelPlugin.js

```diff
@@ -0,0 +1,586 @@
+'use strict';
+
+const esutils = require('esutils');
+
+function helper(babel, opts) {
+  const {types: t} = babel;
+
+  const visitor = {};
+
+  visitor.JSXNamespacedName = function(path, state) {
```

Comment:
Why write it this way, I wonder? 🤔

```jsx
const visitor = {
  JSXNamespacedName(path, state) {
    // ...
  },
  JSXSpreadChild(path) {
    // ...
  }
  // ...
}
```

Reply:
Ah, I see, this is a fork of https://github.com/babel/babel/blob/master/packages/babel-helper-builder-react-jsx/src/index.js